distributed/client.py

```diff
@@ -3933,9 +3949,15 @@ def get_task_stream(
             source, figure = task_stream_figure(sizing_mode="stretch_both")
             source.data.update(rects)
             if plot == "save":
-                from bokeh.plotting import save
-
-                save(figure, title="Dask Task Stream", filename=filename)
+                from bokeh.plotting import save, output_file
+
+                output_file(filename=filename, title="Dask Task Stream")
+                save(
+                    figure,
+                    title="Dask Task Stream",
+                    filename=filename,
+                    bokeh_resources=bokeh_resources,
```

Comment:
Here we should just use `resources=` in bokeh's `save` function:

```suggestion
                    resources=bokeh_resources,
```